### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by saicharan-gouru**
-Added basic routes.
-Added Navbar.
-Added Homepage.
-Added Footer.

quiz homepage

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
+<head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
+    <meta name="description" content="Web site created using create-react-app" />
```
Comment (junaidshaikh-js) [inline public/index.html:9]: Provide a good description for you website.

File: public/index.html [hunk 2]
```diff
@@ -25,8 +23,9 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
```
Comment (junaidshaikh-js) [inline public/index.html]: Provide a good title for the website

File: src/components/Footer/Footer.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import "./Footer.css";
+import { AiFillLinkedin } from 'react-icons/ai';
+import { AiFillTwitterCircle} from 'react-icons/ai';
+import {AiFillGithub} from 'react-icons/ai';
```
Comment (junaidshaikh-js) [inline src/components/Footer/Footer.js]: All the icons can be imported in the single line 

`import { this, that } from 'react-icons/ai';`

File: src/components/index.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+import { Navbar } from "./Navbar/Navbar";
+import { Footer } from "./Footer/Footer";
+
+export { Navbar, Footer };
```
Comment (junaidshaikh-js) [inline src/components/index.js:4]: You can directly export components

```
export { Navbar } from './Navbar/Navbar';
```

File: src/pages/index.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+import { Homepage } from "./homepage/Homepage";
+
+export { Homepage };
```
Comment (junaidshaikh-js) [inline src/pages/index.js:3]: Here also the component can be directly exported

**Review by junaidshaikh-js — COMMENTED** [review]
Good work! 

Suggested some changes, everything else looks fine.

File: src/index.js [hunk 6]
```diff
@@ -3,10 +3,13 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
```
Comment (ranitasahaa1230) [inline src/index.js]: Remove all these files and this line too. (Here its line no 5)

**Review by ranitasahaa1230 — APPROVED** [review]
Good use of semantic tags
Looks good to merge

**Pull request merged**